packages/utils/src/streams.ts

```diff
@@ -1,25 +1,17 @@
 import {Stream} from 'stream'
-
 import pipe from 'pump'
-export {pipe}
-
 import through from 'through2'
-export {through}
-
-export {default as parallel} from 'parallel-transform'
-
-export {default as from} from 'from2'
-
+import parallel from 'parallel-transform'
+import from from 'from2'
 import pumpify from 'pumpify'
 
-// Bad types
 type PumpifyFn = (...streams: Stream[]) => pumpify
-// const pipeline = (pumpifyFn as any) as PumpifyFn & {obj: PumpifyFn}
+
 const pipeline = (pumpify.ctor({
   autoDestroy: false,
   destroy: false,
   objectMode: true,
   highWaterMark: 160,
 }) as any) as PumpifyFn
 
-export {pipeline}
+export const streams = {pipe, through, parallel, from, pipeline}
```

Comment:
Each one of these is a stream helper. Any chance we can't just export each of these individually? 

```
import {through} from '@blitzjs/utils'
```

Maybe namespacing them might make sense if we are concerned with execution context.

```
import {through} from '@blitzjs/utils/streams'
```

Reply:
Happy to change this but I think it makes it simpler for each module to have a single export–similar to how the logger now just exports `log` instead of doing `import { error } from ...`. Thoughts?

Reply:
Hmm spider dependencies like this need to be handled with care as they become hard to refactor the more they are used. Also maybe this is bikeshedding so tell me to shut up if so but whatever we decide here will probably stay.

There are tradeoffs on both sides:
* `streams.through.obj(...)` Provides more context as to what it is is. (through is just a bad name - although it is a standard and well known module in Node)
* Grouping by namespace (ie. streams) uses less global utils namespace by compacting to a standard lib
* It makes sense to consider our view rendering layer as a single dependency because it is somewhat application specific and will likely contain app specific things such as ascii logos or special layouts and standard colours etc. 
 
However
* None of the stuff within `streams` depends on each other they are all top level helpers that happen to deal with node streams and could be used in any node project.
* By using a named `streams` export the `streams` variable declaration becomes unusable where it would be handy to call let's say a variable containing an array of streams `streams`. Perhaps `streamUtils` might be more accurate but I really think that deep imports are the best way to manage this `@blitzjs/utils/streams` as it also allows destructured imports.
* It is common to pipeline nested pipelines together:
```
pipeline(inp, pipeline(a, b, c), pipe(d, e))
```
With namespacing and without aliasing this gets messy:
```
streams.pipeline(inp, streams.pipeline(a, b, c), streams.pipe(d, e))
```
* If we get the namespacing wrong it is hard to fix up later so using single level namespacing or context level namespacing is one less decision we need to make. 

My preference is for deep imports.